docs/release-process.md

```diff
@@ -0,0 +1,173 @@
+# How to release c2pa-rs and related crates
+
+The build process for the crates in this repo is automated using [release-plz](https://release-plz.dev), a Rust-specific CI tool. Release-plz is two components: a Rust-based command-line tool ([GitHub](https://github.com/release-plz/release-plz)) and a pre-packaged GitHub Actions wrapper ([GitHub](https://github.com/release-plz/action)). We use the pre-packaged GitHub Actions wrapper in this project. Colin and Eric have both submitted fixes to release-plz. The developer is generally responsive and PRs are typically merged and released within a week or two.
+
+Release-plz leverages information contains in the various `Cargo.toml` files and its own [`release-plz.toml`](../release-plz.toml) configuration file. In this document, we will talk only about how it release-plz is configured for this project. For more detail about the available settings in release-plz, we recommend reading the [release-plz documentation site](https://release-plz.dev/docs), which is quite extensive.
+
+## How it works
+
+We use the pre-packaged [release-plz GitHub Actions wrapper](https://github.com/release-plz/action) to trigger builds. This is configured in our [`release.yml`](../.github/workflows/release.yml) task.
+
+For additional information, please read the [release-plz documentation for GitHub Action](https://release-plz.dev/docs/github/quickstart).
+
+In this repo, the build automation is triggered for each commit to the `main` branch.
+
+This task runs using a GitHub access token tied to Eric's `@scouten-adobe` account, which is why all PRs and releases are under Eric's name. (See ticket CAI-7621 for moving this to a team-owned bot account.)
+
+### Create a release pull request
+
+This runs [`release-plz release-pr`](https://release-plz.dev/docs/usage/release-pr). For each published crate in the repo, it looks for certain types of commits since the last tagged release _within_ the source tree for the crate. The exact list of commit types that trigger a release PR is configured by the `release-commits` section of [`release-plz.toml`](../release-plz.toml), but notably is set to ignore `chore` commits and include `feat`, `fix`, and `docs` commits. If it finds any matching, it will open a new release pull request or update the existing release. (NOTE: When Colin's [PR #2196 to release-plz](https://github.com/release-plz/release-plz/pull/2196) lands there will be a new trigger here which will reference intra-project dependency updates even if the downstream project has no commits of its own.)
```

Comment:
Apparently the full list includes `update` though I'm not sure when we would use that:
release_commits = "^(feat|fix|update|doc)[(:]"
Also it seems like 'security' should trigger something.